GitIQT/Program.cs

```diff
@@ -11,6 +11,7 @@ static void Main(string[] args)
                 "======================================================================================================================",
                 "It's your first day at your new job and there's a lot of stuff to do. Your boss gives you a list of tasks to complete:",
                 "1. Clone a repository",
+                "    ( Clone from main branch or a specific branch ) ",
```

Comment:
what was the intent for this? The formatting seems off

Reply:
Intent was to provide multiple options expect in the initial prompt, supposed to give the appearance of being indented thus still related to the 1st "Clone a repository" task. as the expectations on Issue #1 asked for cloning a generic branch (default master/main) or cloning from a specific branch